client/platform/web-girder/store/Brand.ts

```diff
@@ -35,6 +35,8 @@ const brandModule: Module<BrandState, RootState> = {
         at 2AM EST/EDT on Thursdays. Downtime is typically
         less than 10 minutes.`,
       alertMessage: '',
+      trainingMessage: `This server is updated on Thursday at 2AM EST.
+      If your training job is running at that time it may be restarted or killed.`,
```

Comment:
Does this still happen? Should we have a different default? Also do we need to update this elsewhere if this isn't happening?

Reply:
We could set a different or empty default now that we don't do it.  Something about training can take a long time depending on the configuration and the number and size of the datasets.